include/nanogui/screen.h

```diff
@@ -123,7 +123,11 @@ class NANOGUI_EXPORT Screen : public Widget {
     virtual bool keyboardCharacterEvent(unsigned int codepoint);
 
     /// Window resize event handler
-    virtual bool resizeEvent(const Vector2i &) { return false; }
+    virtual bool resizeEvent(const Vector2i& size) { if (mResizeCallback) mResizeCallback(size); return false; }
```

Comment:
One minor request: can you move the implementation of the virtual function to the ``.cpp`` file?